app/controllers/Search.scala

```diff
@@ -48,4 +56,92 @@ object Search extends Controller {
       }
     }
   }
+
+  def openSearchDescription(dtype: String) = Action { implicit request =>
+    def templateUrl(ftype: String) = {
+      routes.Search.index.absoluteURL + "/results?q={searchTerms}&page={startPage?}&perpage={count?}" +
+      "&target=" + dtype + "&format=" + ftype
+    }
+    Ok(
+      <OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/">
```

Comment:
Might be - our current practice (e.g. SWORD, Mets) for XML docs is not to use the view templates - should they be refactored as well?